ui/src/app/services/error-reporter.service.ts

```diff
@@ -35,6 +35,18 @@ export class ErrorReporterService extends ErrorHandler {
     if (!this.stackdriverReporter) {
       return;
     }
+
+    // xhrError is set to true by the interceptor. The rejection piece handles
```

Comment:
nit: one of these comments should note **why** we're trying to suppress error reporting of XHR errors.